src/config/index.ts

```diff
@@ -103,37 +97,47 @@ export default {
     dbName: process.env.DATABASE || 'readitla_ril-tmp',
     tz: process.env.DATABASE_TZ || 'US/Central',
   },
-  sentry: {
-    dsn: process.env.SENTRY_DSN || '',
-    release: process.env.GIT_SHA || '',
-    environment: process.env.NODE_ENV || 'development',
+  dataloaderDefaults: {
```

Comment:
this is the only real addition to this file in this PR. The rest is putting the config values in alphabetical order (at the top level) to understand better what exists vs doesn't.